LaTeX/LaTeX.sublime-syntax

```diff
@@ -446,7 +446,7 @@ contexts:
     - match: \$
       scope: string.other.math.latex punctuation.definition.string.begin.latex
       push:
-        - meta_scope: meta.environment.math.latex
+        - meta_scope: meta.environment.math.latex meta.environment.math.inline.latex
```

Comment:
You don't need to provide both, `meta.environment.math.latex` and `meta.environment.math.block.latex`. The more specific one should be enough for 99.9% of all cases.